src/hooks/useNotificationTypes.tsx

```diff
@@ -205,9 +206,72 @@ export const notificationTypes: NotificationTypeConfig[] = [
       return () => {};
     },
   },
+  {
+    type: NotificationType.ReleaseUpdates,
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+      const body = (
+        <div>
+          {stringGetter({
+            key: 'NOTIFICATIONS.RELEASE_REWARDS_AND_FULL_TRADING.BODY',
```

Comment:
not sure what to do in order to allow us to just use `STRING_KEYS.RELEASE_REWARDS_AND_FULL_TRADING_BODY`

the double `NOTIFICATIONS` is throwing me off 😢